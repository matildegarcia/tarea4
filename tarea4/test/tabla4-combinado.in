# creo una tabla de tamanio 10
crearTTablaJugadores 10

# creo 10 jugadores, los inserto en la tabla, verifico que pertenezcan y los obtengo
# 1
crearPartida

crearJugada
3 1 2 3 1
agregarEnPartida

crearJugada
4 1 2 3 4
agregarEnPartida

crearJugador 1 11 Uruguay Jugador1
crearFecha 1/11/2022
insertarJugadorEnTabla

perteneceATTablaJugadores Jugador1
obtenerJugadorDeTTablaJugadores Jugador1

# 2
crearPartida

crearJugada
4 1 2 3 1
agregarEnPartida

crearJugada
2 1 3 4 2
agregarEnPartida

crearJugador 2 22 Uruguay Jugador2
crearFecha 1/11/2011

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador2
obtenerJugadorDeTTablaJugadores Jugador2

# 3
crearPartida

crearJugada
1 4 1 4 1
agregarEnPartida

crearJugada
4 1 2 3 4
agregarEnPartida

crearJugador 3 33 Uruguay Jugador3
crearFecha 2/11/2001

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador3
obtenerJugadorDeTTablaJugadores Jugador3

# 4
crearPartida

crearJugada
3 1 2 3 1
agregarEnPartida

crearJugada
4 1 2 3 4
agregarEnPartida

crearJugador 4 44 Uruguay Jugador4
crearFecha 4/11/1999

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador4
obtenerJugadorDeTTablaJugadores Jugador4

# 5
crearPartida

crearJugada
1 1 2 3 1
agregarEnPartida

crearJugada
4 1 1 3 1
agregarEnPartida

crearJugador 5 55 Uruguay Jugador5
crearFecha 5/1/1998

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador5
obtenerJugadorDeTTablaJugadores Jugador5

# 6
crearPartida

crearJugada
1 2 2 3 1
agregarEnPartida

crearJugada
2 1 2 2 4
agregarEnPartida

crearJugador 6 66 Uruguay Jugador6
crearFecha 6/12/1997

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador6
obtenerJugadorDeTTablaJugadores Jugador6

# 7
crearPartida

crearJugada
3 1 2 3 1
agregarEnPartida

crearJugada
4 1 2 3 4
agregarEnPartida

crearJugador 7 77 Uruguay Jugador7
crearFecha 7/7/1996

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador7
obtenerJugadorDeTTablaJugadores Jugador7

# 8
crearPartida

crearJugada
3 3 3 3 3
agregarEnPartida

crearJugada
4 4 4 4 4
agregarEnPartida

crearJugador 8 88 Uruguay Jugador8
crearFecha 1/11/1995

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador8
obtenerJugadorDeTTablaJugadores Jugador8

# 9
crearPartida

crearJugada
2 1 1 3 1
agregarEnPartida

crearJugada
1 4 2 3 4
agregarEnPartida

crearJugador 9 19 Uruguay Jugador9
crearFecha 1/11/1994

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador9
obtenerJugadorDeTTablaJugadores Jugador9

# 10
crearPartida

crearJugada
1 1 1 1 1
agregarEnPartida

crearJugada
5 2 2 4 4
agregarEnPartida

crearJugador 10 10 Uruguay Jugador10
crearFecha 1/11/1993

insertarJugadorEnTabla
perteneceATTablaJugadores Jugador10
obtenerJugadorDeTTablaJugadores Jugador10

# imprimo la tabla
imprimirTTablaJugadores


# elimino los jugadores de id par y verifico que no pertenecen
eliminarJugadorDeTTablaJugadores Jugador2
perteneceATTablaJugadores Jugador2
eliminarJugadorDeTTablaJugadores Jugador4
perteneceATTablaJugadores Jugador4
eliminarJugadorDeTTablaJugadores Jugador6
perteneceATTablaJugadores Jugador6
eliminarJugadorDeTTablaJugadores Jugador8
perteneceATTablaJugadores Jugador8
eliminarJugadorDeTTablaJugadores Jugador10
perteneceATTablaJugadores Jugador10

# imprimo la tabla 
imprimirTTablaJugadores

# libero la tabla
liberarTTablaJugadores

Fin