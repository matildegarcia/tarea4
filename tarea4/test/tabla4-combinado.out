1>#  creo una tabla de tamanio 10.
2>3>#  creo 10 jugadores, los inserto en la tabla, verifico que pertenezcan y los obtengo.
4>#  1.
5>6>7>8>9>10>11>12>13>El jugador con nombre Jugador1 está en la tabla.
14>Jugador 1: Jugador1, Uruguay, 11 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
15>#  2.
16>17>18>19>20>21>22>23>24>El jugador con nombre Jugador2 está en la tabla.
25>Jugador 2: Jugador2, Uruguay, 22 años
Jugada: 2 
Movimientos: Derecha Arriba Abajo Izquierda 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Derecha 
26>#  3.
27>28>29>30>31>32>33>34>35>El jugador con nombre Jugador3 está en la tabla.
36>Jugador 3: Jugador3, Uruguay, 33 años
Jugada: 1 
Movimientos: Abajo Derecha Abajo Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
37>#  4.
38>39>40>41>42>43>44>45>46>El jugador con nombre Jugador4 está en la tabla.
47>Jugador 4: Jugador4, Uruguay, 44 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
48>#  5.
49>50>51>52>53>54>55>56>57>El jugador con nombre Jugador5 está en la tabla.
58>Jugador 5: Jugador5, Uruguay, 55 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Derecha Arriba Derecha 
59>#  6.
60>61>62>63>64>65>66>67>68>El jugador con nombre Jugador6 está en la tabla.
69>Jugador 6: Jugador6, Uruguay, 66 años
Jugada: 1 
Movimientos: Izquierda Izquierda Arriba Derecha 
Jugada: 2 
Movimientos: Derecha Izquierda Izquierda Abajo 
70>#  7.
71>72>73>74>75>76>77>78>79>El jugador con nombre Jugador7 está en la tabla.
80>Jugador 7: Jugador7, Uruguay, 77 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
81>#  8.
82>83>84>85>86>87>88>89>90>El jugador con nombre Jugador8 está en la tabla.
91>Jugador 8: Jugador8, Uruguay, 88 años
Jugada: 3 
Movimientos: Arriba Arriba Arriba Arriba 
Jugada: 4 
Movimientos: Abajo Abajo Abajo Abajo 
92>#  9.
93>94>95>96>97>98>99>100>101>El jugador con nombre Jugador9 está en la tabla.
102>Jugador 9: Jugador9, Uruguay, 19 años
Jugada: 1 
Movimientos: Abajo Izquierda Arriba Abajo 
Jugada: 2 
Movimientos: Derecha Derecha Arriba Derecha 
103>#  10.
104>105>106>107>108>109>110>111>112>El jugador con nombre Jugador10 está en la tabla.
113>Jugador 10: Jugador10, Uruguay, 10 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugada: 5 
Movimientos: Izquierda Izquierda Abajo Abajo 
114>#  imprimo la tabla.
115>Jugador 6: Jugador6, Uruguay, 66 años
Jugada: 1 
Movimientos: Izquierda Izquierda Arriba Derecha 
Jugada: 2 
Movimientos: Derecha Izquierda Izquierda Abajo 
6/12/1997
Jugador 7: Jugador7, Uruguay, 77 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
7/7/1996
Jugador 8: Jugador8, Uruguay, 88 años
Jugada: 3 
Movimientos: Arriba Arriba Arriba Arriba 
Jugada: 4 
Movimientos: Abajo Abajo Abajo Abajo 
1/11/1995
Jugador 9: Jugador9, Uruguay, 19 años
Jugada: 1 
Movimientos: Abajo Izquierda Arriba Abajo 
Jugada: 2 
Movimientos: Derecha Derecha Arriba Derecha 
1/11/1994
Jugador 10: Jugador10, Uruguay, 10 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugada: 5 
Movimientos: Izquierda Izquierda Abajo Abajo 
1/11/1993
Jugador 1: Jugador1, Uruguay, 11 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
1/11/2022
Jugador 2: Jugador2, Uruguay, 22 años
Jugada: 2 
Movimientos: Derecha Arriba Abajo Izquierda 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Derecha 
1/11/2011
Jugador 3: Jugador3, Uruguay, 33 años
Jugada: 1 
Movimientos: Abajo Derecha Abajo Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
2/11/2001
Jugador 4: Jugador4, Uruguay, 44 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
4/11/1999
Jugador 5: Jugador5, Uruguay, 55 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Derecha Arriba Derecha 
5/1/1998
116>#  elimino los jugadores de id par y verifico que no pertenecen.
117>118>El jugador con nombre Jugador2 NO está en la tabla.
119>120>El jugador con nombre Jugador4 NO está en la tabla.
121>122>El jugador con nombre Jugador6 NO está en la tabla.
123>124>El jugador con nombre Jugador8 NO está en la tabla.
125>126>El jugador con nombre Jugador10 NO está en la tabla.
127>#  imprimo la tabla .
128>Jugador 7: Jugador7, Uruguay, 77 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
7/7/1996
Jugador 9: Jugador9, Uruguay, 19 años
Jugada: 1 
Movimientos: Abajo Izquierda Arriba Abajo 
Jugada: 2 
Movimientos: Derecha Derecha Arriba Derecha 
1/11/1994
Jugador 1: Jugador1, Uruguay, 11 años
Jugada: 3 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
1/11/2022
Jugador 3: Jugador3, Uruguay, 33 años
Jugada: 1 
Movimientos: Abajo Derecha Abajo Derecha 
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
2/11/2001
Jugador 5: Jugador5, Uruguay, 55 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Derecha 
Jugada: 4 
Movimientos: Derecha Derecha Arriba Derecha 
5/1/1998
129>#  libero la tabla.
130>131>Fin.
